index.html

```diff
@@ -3518,6 +3492,30 @@ <h2>
 ]»
         </pre>
 
+        <p>
+The next example demonstrates a JSON-encoded metadata structure that might be
+used as <a href="#did-document-metadata-properties">DID document metadata</a> to
+describe timestamps of the DID document.
+        </p>
+
+        <pre class="example" title="JSON-encoded DID document metadata example">
+{
+  "created": "2002-10-10T17:00:00Z",
```

Comment:
I'm wondering if we should use a date that is within the range of when the DID spec existed. So, perhaps using created dates after 2018 and update dates after 2020 would be more realistic? :)

Reply:
Updated the example dates in https://github.com/w3c/did-core/pull/365/commits/260972366213973017c1b8bdba2f49d250de7ed5.